src/network/hardware-benchmark.md

```diff
@@ -0,0 +1,214 @@
+# Hardware Benchmark
+
+## Lab setup
+
+`Device` Device being tested  
+
+`Endpoint1`, `Endpoint2` Devices not limited by CPU or network.
+
+![image](images/hardware-benchmark-network.jpg)
+
+Instructions below are non-persistent. When device is restarted changes will be removed. `wireguard` package must be installed.
+
+### Device Lab Configuration
+
+**Configure interfaces**
```

Comment:
```suggestion
#### Configure interfaces
```
Having these as proper subsections is more idiomatic and will make them appear in the ToC.